Rakefile

```diff
@@ -50,6 +50,23 @@ namespace :arm do
         FileUtils.rm_rf(folder_to_be_cleaned)
       end
     end
+
+    Dir.chdir(File.expand_path('../data', __FILE__))
```

Comment:
could we create a method we call instead of repeating the code from above and passing "data" folder instead of "management"?